### DISCUSSION TITLE
Dagster and Meltano via Dagster pipes?

### DISCUSSION THREAD

**Discussion by seunggs** [dagster-io/dagster#18779] [Q&A] [answered]
Hi, I’m going through the tutorial for dagster/meltano using this article: https://dagster.io/blog/dagster-meltano-integration-tutorial. 

My question is, now that Dagster released `pipes`, is that the recommended way of running something like Meltano?

**Comment by yuhan** [comment, marked as answer]
You may customize a pipes protocol with Meltano: https://docs.dagster.io/guides/dagster-pipes/dagster-pipes-details-and-customization

**Reply by Jcohen010** [reply]
@seunggs have you toyed around with this yet? I'm a meltano user and am not a huge fan of the current dagster-meltano plugin, particularly that it's not currently possible to build assets out of meltano tap streams. I'm working on contributing to the project to add this functionality to the plug-in.

**Reply by seunggs** [reply]
@Jcohen010 Sorry no - haven't had the time so using the plugin for now. If you're coming up with an alternative solution though, definitely interested in seeing!
